src/hubcast/clients/github/client.py

```diff
@@ -110,3 +137,43 @@ async def get_prs(self, branch=None):
                 url = f"{url}?head={self.repo_owner}:{branch}"
                 prs_res = await gh.getitem(url)
                 return [pr["number"] for pr in prs_res]
+
+    async def post_comment(self, issue_number: int, body: str):
+        payload = {"body": body}
+
+        gh_token = await self.auth.authenticate_installation(
+            self.repo_owner, self.repo_name
+        )
+
+        async with aiohttp.ClientSession() as session:
+            gh = gh_aiohttp.GitHubAPI(session, self.requester, oauth_token=gh_token)
+
+            url = f"/repos/{self.repo_owner}/{self.repo_name}/issues/{issue_number}/comments"
+            await gh.post(url, data=payload)
+
+    async def react_to_comment(self, comment_id: int, content: str):
+        # Valid content examples: "+1", "-1", "laugh", "hooray", "confused", "heart", "rocket", "eyes"
+        payload = {"content": content}
```

Comment:
Let's add a docstring to this method that describes the valid examples and method usage. In addition I'd add all of these valid examples to a list and then check that the content string is in the allowed responses and raise an error if it is not valid. Maybe also name the parameter "reaction" instead of content?